docker-compose.dev.yml

```diff
@@ -44,3 +44,19 @@ services:
     restart: "no"
   uri-info:
     restart: "no"
+  delta-producer-bg-jobs-initiator-administrative-units:
```

Comment:
I think the `file` and `delta-producer-dump-file-publisher` services are still missing?